src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -49,33 +46,36 @@ export const ExecutiveFirstPage = ({ data }) => {
-    ramOccuranceTableData.push(...condtionsInfo.memory.occurances);
-    cpuOccuranceTableData.push(...condtionsInfo.cpu.occurances);
+    ioOccuranceTableData.push(...condtionsInfo.io.occurrences);
+    ramOccuranceTableData.push(...condtionsInfo.memory.occurrences);
+    cpuOccuranceTableData.push(...condtionsInfo.cpu.occurrences);
 
     return <Fragment key="first-page">
         <Text>
             This executive summary highlights the performance for your registered systems included in the resource optimization service.
         </Text>
-        <Text>
-            {`This report gives you an overview of:${newLine}`}
-        </Text>
-
-        <Text>
-            {
-                `${bullet} The number of systems registered
-                ${bullet} Number of registered systems in a non-optimal state
-                ${bullet} Number of system performance issues
-                ${bullet} Description of system performance levels
-                ${bullet} Performance level details for system resources operating in a non-optimal state`}
-        </Text>
 
         {/* {Total Systems} */}
         <Text style={styles.execHeading}>Registered systems</Text>
         {/* eslint-disable-next-line max-len */}
-        <Text>{`There are ${totalCount} systems registered in the resource optimization service. The service identified ${optimizedCount} of ${totalCount} systems as optimized, and ${nonOptimizedCount} of ${totalCount} registered systems as having a non-optimal state.`}</Text>
+        <Text>
+            {/* eslint-disable-next-line max-len */}
+            <Text>{`There are`}</Text><Text style={styles.bold}>{` ${totalCount} registered systems `}</Text><Text>{`in the resource optimization service.${newLine}`}</Text>
+            {/* eslint-disable-next-line max-len */}
+            <Text style={styles.bold}>{`${optimizedCount}`}</Text><Text>{` of ${totalCount} systems are identified as `}</Text><Text style={styles.bold}>optimized, </Text>
+            {/* eslint-disable-next-line max-len */}
+            <Text style={styles.bold}>{`${nonOptimizedCount}`}</Text><Text>{` of ${totalCount} systems as having a `}</Text><Text style={styles.bold}>non-optimal</Text><Text>{` state.${newLine}`}</Text>
+            <Text style={styles.bold}>{`${staleCount}`}</Text><Text>{` of ${totalCount} systems are `}</Text><Text style={styles.bold}>stale*</Text>
+        </Text>
+
+        <Text style={styles.execInfoText}>Suggestions for stale systems might no longer apply due to systems not being refreshed in 7 days.*</Text>
 
         <Text style={styles.execHeading}>Breakdown of registered systems</Text>
+
+        {/* TODO: update based on api response */}
+        {/* eslint-disable-next-line max-len */}
+        <Text>{`6 systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information Enabled. That helps us provide better suggestions either in breakdown of occurance or registered system section`}</Text>
```

Comment:
As mentioned by @r14chandra about incorrect plural form `systems` when having one system, I think you would also need to take care of `has/have` conditionally?
Also change `occurance` to `occurrence`

Reply:
correct @upadhyeammit I discussed the same with Stefan and updated the [jira card description](https://issues.redhat.com/browse/RHIROS-687) based on it. Also added a pluralize method to fix the same!

Reply:
```suggestion
        <Text>{`6 systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information Enabled. That helps us provide better suggestions either in breakdown of occurrence or registered system section`}</Text>
```

Reply:
Thanks @kgaikwad I have fixed the typo for same in the latest commit.